README.md

```diff
@@ -1267,21 +1268,21 @@ trivy:
   allow_failure: true
   script:
     # Build image
-    - docker build -t trivy-ci-test:$CI_COMMIT_SHA .
+    - docker build -t $IMAGE .
     # Build report
-    - ./trivy --exit-code 0 --cache-dir $CI_PROJECT_DIR/.trivycache/ --no-progress  --format template --template "@contrib/gitlab.tpl" -o ${CI_PROJECT_DIR}/gl-container-scanning-report.json trivy-ci-test:$CI_COMMIT_SHA
+    - ./trivy --exit-code 0 --cache-dir $CI_PROJECT_DIR/.trivycache/ --no-progress --format template --template "@contrib/gitlab.tpl" -o ${CI_PROJECT_DIR}/gl-container-scanning-report.json $IMAGE
```

Comment:
Can we drop ${CI_PROJECT_DIR} here as well?

Reply:
@tnir Let me confirm why the following works since I'm not familiar with GitLab. 
```
  artifacts:
    reports:
      container_scanning: gl-container-scanning-report.json
```

We need to specify a relative path to the project directory ($CI_PROJECT_DIR) as `artifacts:reports:container_scanning`, right?
https://docs.gitlab.com/ee/ci/yaml/#artifactspaths

Reply:
As the working directory in CI execution environment is always `${CI_PROJECT_DIR}` unless `cd [somewhere]`, we can drop `${CI_PROJECT_DIR}` under `.script` as well.

We can drop`${CI_PROJECT_DIR}` For cache directory, too. @mrueg 

> We need to specify a relative path to the project directory ($CI_PROJECT_DIR) as artifacts:reports:container_scanning, right?

Yes. But I'm not sure if it still work when the absolute path is specified. Relative path is enough to work and is simpler than absolute path. @knqyf263

Reply:
I understand. Thank you for explanation.